code/Python/sep/sep_aq.py

```diff
@@ -12,15 +14,22 @@
 FTP_USERNAME = 'FTP_USERNAME'
 FTP_PASSWORD = 'FTP_PASSWORD'
 
-WORK_DIRECTORY = "../../../work/"
 USE_LOCAL_FILES = True
 
 REF_YEAR = '2019'
 
-with open('../sep/sep.conf.json') as conf_str:
-    conf = json.load(conf_str)
 
-logging.basicConfig(filename=WORK_DIRECTORY + 'sep-aq.log', encoding='utf-8', level=logging.DEBUG)
+def get_working_directory():
```

Comment:
The `conf` dictionnary used in this function must probably be passed as a parameter.